articles/topics/identity-glossary.md

```diff
@@ -6,6 +6,10 @@ articles:
   - tokens/jwt
 ---
 
+::: panel-info Auth0 Identity Glossary
+For addition definitions of Auth0-related terms, please see the [Auth0 Identity Glossary](https://auth0.com/identity-glossary).
+:::
```

Comment:
I was thinking the same, but for now, I think we should link since those docs are in the backlog and it would be nice to have a place to go to get that info. I see a lot of defs questions asked in support.